src/Miki/Modules/Accounts/AccountsModule.cs

```diff
@@ -1033,30 +1033,29 @@ await e.ErrorEmbedResource("give_error_amount_unparsable")
         [Command("daily")]
         public async Task GetDailyAsync(IContext e)
         {
-            var context = e.GetService<MikiDbContext>();
-
+            MikiDbContext context = e.GetService<MikiDbContext>();
             User u = await DatabaseHelpers.GetUserAsync(context, e.GetAuthor());
 
-            if(u == null)
+            if(u == null) // Check it the user exists.
             {
                 await e.ErrorEmbed(e.GetLocale().GetString("user_error_no_account"))
                     .ToEmbed().QueueAsync(e, e.GetChannel());
                 return;
             }
 
+            /* Base variables */
             int dailyAmount = 100;
             int dailyStreakAmount = 20;
 
-            if(await u.IsDonatorAsync(context))
+            if (await u.IsDonatorAsync(context)) // Check donator status.
```

Comment:
Please use `IsDonatorAsync(...).ConfiguresAwait(false);` And also please look at the donator module for a more up to date approach on checking if a user is a donator.